### ISSUE TITLE
Instant Reload (for energy-based weapons)

### ISSUE THREAD

**Issue by SylveonBottle** [nullworks/cathook#183]
I can confirm from #182 that taunting with removecond will instantly refill your ammo. This lets you spam the shit out of rockets, or instantly reload a fully charged rocket.

Now requesting for automation to auto-taunt when weapons like The Cow Mangler 5000 is empty on charge. Autotaunt + Remove Taunting = cow mangler ammo hack or some shit.

**Comment by SylveonBottle** [comment]
It also seems to instant-reload if you removecond while mid-air after a rocket jump. Doesn't seem to work for normal jumping.

**Comment by gir489** [comment]
Rocket jumping adds the TFCondEx2_BlastJumping condition to you. I also noticed that FaN jumping adds the TFCond_Stunned condition, too. [Use this](https://github.com/alliedmodders/sourcemod/blob/master/plugins/include/tf2.inc#L74) as a basis for comparison. The TFConds I released on UC like 4 years ago are slightly out of date. They're missing some new additions, but for the most part, are on the up-and-up.

**Comment by Blackfire62** [comment]
It indeed works, done this some time ago: https://www.youtube.com/watch?v=0z8T0CWPlFQ
Start taunting, and after you got the TFCond_Taunting you can cond reset and continue.

**Comment by gir489** [comment]
@Blackfire62 Can you send me the Glock.vpk you have? I lost it during a format.

**Comment by Blackfire62** [comment]
@gir489 here you go, uploaded on some random website https://files.fm/u/8m4mv6sn

**Comment by gir489** [comment]
@Blackfire62 Ayy, thanks mate. For S&G where did you find it? I don't remember where I released it...

**Comment by Blackfire62** [comment]
@gir489 no problem, I found it on UnknownCheats im pretty sure, when I was checking out few of your old threads.

**Comment by BenCat07** [comment]
Removecond was pretty much patched

**Comment by BenCat07** [comment]
this can't be brought back sadly, besides if you manage to not have used your weapon for x seconds where x is the standard clip size without switching weapons or holding M1/M2/Reload key


TL;DR

Only useable on cl_autoreload 0
